data/csv/uk_cities.csv

```diff
@@ -0,0 +1,37 @@
+"Elgin, Scotland, the UK",57.653484,-3.335724
```

Comment:
Where does this file originate from?

Reply:
I'm trying to remember ... if I can't find it, I will download fresh from here:

https://simplemaps.com/data/world-cities

The free download here uses the Creative Commons Attribution License.

Reply:
Can you write a LICENSE file and remark that in there?

Reply:
@xhochy For now I removed the files that are not used, and I manually created a new uk_cities.csv with just two cities and looked up lat/lon online, so this is safe to merge.

Once the DataFusion donation is accepted I'll create some better test files and delete the trivial ones. I don't want to complicate the donation by doing that now and updating all the tests.